src/Training.AirBnb.Clone.Frontend/src/common/views/HomeView.vue

```diff
@@ -6,7 +6,7 @@
   <main-header/>
   
   <!-- Listings Container -->
-  <listings-container/>
+  <listings-container class="mt-44"/>
```

Comment:
Isn't it working if you give the margin to a main div inside the ListingContaineer?

Reply:
of course, it is work

Reply:
Fine